controller/engine_controller.go

```diff
@@ -1017,13 +1021,13 @@ func checkSizeBeforeRestoration(engine *longhorn.Engine, ds *datastore.DataStore
 		} else if bvSize > v.Spec.Size {
 			// TODO: Find a way to update volume.Spec.Size outside of the controller
 			// The volume controller will update `engine.Spec.VolumeSize` later then trigger expansion call
-			logrus.Infof("engine monitor: Prepare to expand the size from %v to %v for DR volume %v", v.Spec.Size, bvSize, v.Name)
+			log.Infof("Prepare to expand the size from %v to %v for DR volume %v", v.Spec.Size, bvSize, v.Name)
 			v.Spec.Size = bvSize
 			if _, err := ds.UpdateVolume(v); err != nil {
 				if !datastore.ErrorIsConflict(err) {
 					return false, err
 				}
-				logrus.Debugf("engine monitor: Retrying update the volume %v size before restore", v.Name)
+				log.Debugf("Retrying size update for DR volume %v before restore", v.Name)
```

Comment:
Can we use `WithField("volume", e.Spec.VolumeName)`? Similar below.

Reply:
yes, we can add it as a field.